content/en/community/squads.md

```diff
@@ -0,0 +1,76 @@
+--- 
+title: "CHT Squads" 
+linkTitle: "Community"
+weight: 2 
+description: > 
+  CHT Initiatives 
+---
+
+  **What is a Squad?**
+
+A CHT Squad is a small, action-oriented team that works together to solve high-priority challenges for the community. These teams bring together developers, designers, partners, and other contributors to build and implement important new features for the CHT ecosystem.
+
+**How Squads Work:**
+
+- Collaborative by design: Anyone can join - whether you're an experienced developer or just getting started
+- Transparent process: Regular meetings, shared notes, and public GitHub repositories keep everyone informed
+- Flexible contributions: Help in whatever way works for you - coding, design, testing, or documentation
+
+**Why Join a Squad?**
+- Work on meaningful projects that make a real difference
+- Learn from and collaborate with other community members
+- Help shape the future of the CHT
+
+
+**The CHT Squad Journey**
+Squads evolve through stages, and anyone can contribute—no matter their experience level. Here are the various stages:
+
+1. **✨** Emerging
```

Comment:
Please review the list of statuses to exactly match the [squad statuses in the roadmap](https://github.com/orgs/medic/projects/112/views/24).